napari/_qt/_tests/test_qt_provide_theme.py

```diff
@@ -22,7 +22,7 @@ def test_provide_theme_hook_registered_correctly(
 ):
     # make a viewer with a plugin & theme registered
     viewer = make_napari_viewer_with_plugin_theme(
-        make_napari_viewer,
+        make_napari_viewer(),
```

Comment:
This bit is new, can you tell me why it was necessary?
I don't totally understand what is happening here, since this test passes without this change in my local dev environment, and also passes in CI runs from previous PRs.

Reply:
This change is just me testing if I could get this test to pass since after multiple retries a job kept failing showing a `Windows fatal exception: access violation`: https://github.com/napari/napari/actions/runs/4885713982/jobs/8721688023#step:8:203

I will revert this change :+1:

Reply:
Got it, thank you!